code/modules/mob/living/say.dm

```diff
@@ -98,8 +98,13 @@ var/list/crit_allowed_modes = list(MODE_WHISPER,MODE_CHANGELING,MODE_ALIEN)
 	if(!message)
 		return
 
+	var/turf/T = get_turf(src)
+
+	var/logged_msg = "\[[time_stamp()]\] [message]"
+	if(T)
+		logged_msg += " ([T.x], [T.y], [T.z])"
 	//Log of what we've said, plain message, no spans or junk
-	say_log += message
+	say_log += logged_msg

```

Comment:
awwww, well I guess I'll remove this and the vote cooldown thing.
You've bested me this time Remie but I'll be back! MWAHAHAHAHAHA